db/migrate/20190504205233_create_topics.rb

```diff
@@ -1,9 +0,0 @@
-class CreateTopics < ActiveRecord::Migration[5.2]
```

Comment:
Probably won't matter for your little app, but this might not end well in the real world (deleting an existing migration). Your existing databases actually have this table and whatever's been added to it. Normally, you'd write a new migration that dropped the table and mark it as unreversible. You can read up on doing that.

Reply:
I shall read into that and do this the right way. Thanks for the tip! 🙏🏻

Reply:
I saw you've added the migration to drop this table, *but* you still need this migration for that to work. Imagine bringing this project up from scratch. When we run, we'll hit the `drop_table` and it'll fail, because this migration isn't there.  Mostly, you'll never want to `git rm` a migration. DM me if you want to talk through this.